bootstrap/kubeadm/api/v1alpha4/kubeadmconfig_types.go

```diff
@@ -18,7 +18,7 @@ package v1alpha4
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha4"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
```

Comment:
these commits says `create core and exp v1beta1 types` but I can only see these import references changes which seems wrong and then are undone here https://github.com/kubernetes-sigs/cluster-api/pull/5248/commits/fa8e80add80739c9f61604426558b61b2520e047#diff-21fc1e037b29befe44af6dafda142f25c8d2ac7d25f149aba46a39a7beccf5fbL21

Should this commit be dropped/conflated with https://github.com/kubernetes-sigs/cluster-api/pull/5248/commits/2ecc83088175283565999dca361bf5c95a093c85 / https://github.com/kubernetes-sigs/cluster-api/pull/5248/commits/fa8e80add80739c9f61604426558b61b2520e047#diff-21fc1e037b29befe44af6dafda142f25c8d2ac7d25f149aba46a39a7beccf5fbL21 ?

Reply:
Some changes got bundled together, I'd rather not go through a rebase process again given the amount of changes that are included in this PR